addons/languages/icelandic/pack/build.gradle

```diff
@@ -0,0 +1,9 @@
+ext.status_icon_text = "is"
+
+ext.dictionaryInputPossibleCharacters = null//"".toCharArray()
+ext.dictionaryInputAdditionalInnerCharacters = null//"".toCharArray()
+ext.shouldGenerateDictionary = true
+ext.shouldBePublished = false
+ext.notPublishingReason = "Initial version has not been published yet"
```

Comment:
These should not be in the `pack`, and also, I think we can publish. Let's try without these

Reply:
ok